components/organisms/TopNav/top-nav.tsx

```diff
@@ -64,7 +66,7 @@ const Nav = ({ className, name = "Main" }: { className?: string; name?: string }
         </li>
         <li>
           <Link
-            className={`text-sm ${getActiveStyle(router.asPath === `/${userInterest}/dashboard/filter/recent`)}`}
+            className={`text-sm ${getActiveStyle(explorePageUrlPattern.test(router.asPath))}`}
```

Comment:
Wouldn't the regex just need to match the pattern in the url below? `/${userInterest}/dashboard/filter/recent`? So something like `\/[^\/]+\/dashboard\/filter\/recent$` for the path?

See the [explainer of the above regex](https://regex101.com/?regex=%5B%5Cp%7BLl%7D0-9-%5D+) on regex101.

Reply:
Nice thoughts 👍 

The changes I made were accounting for all the tab pages around the explore, e.g `reports`, `contributors`, and `activity`. I feel it is ideal to keep the active state on those pages as well. thoughts @open-sauced/engineering?

Reply:
Post PTO, we'll go with what you have proposed as I was suggesting a regex based on what I believe wasn't working prior to this fix.

```
className={`text-sm ${getActiveStyle(router.asPath === `/${userInterest}/dashboard/filter/recent`)}`}
```